src/styles/index.scss

```diff
@@ -1,3 +1,74 @@
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
 }
+
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  position: relative;
+
+  &__center {
+    width: 5vmin;
+    height: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    z-index: 1;
+  }
+
+  &__hand {
+    position: absolute;
+    bottom: 50%;
+    transform-origin: bottom;
+
+    &--minutes {
+      width: 3vmin;
+      height: 20vmin;
+      background-color: #0700ff;
+      animation: rotate-minutes 3600s steps(60) infinite;
+    }
+
+    &--seconds {
+      width: 1.5vmin;
+      height: 38vmin;
+      background-color: #2c8000;
```

Comment:
The color value `#2c8000` should be lowercase to maintain consistency with the code style rules.